src/components/connect/index.tsx

```diff
@@ -80,8 +80,9 @@ const Connect = ({ showWallet, setShowWallet }: any) => {
   useEffect(() => {
     if (typeof window !== 'undefined') {
       if (!window.keplr) {
-        // eslint-disable-next-line no-alert
-        alert('Please install keplr extension');
+        notification.error({
+          message: 'Please install keplr extension',
+        });
       }
```

Comment:
Can we remove this useEffect and instead check if window.keplr is false when the user clicks on the connect button?